Sources/TuistLoader/Models/PackageInfo.swift

```diff
@@ -371,6 +371,7 @@ extension PackageInfo.Target {
 
         /// The name of the build setting.
         public enum SettingName: String, Codable, Hashable {
+            case swiftVersion
```

Comment:
The evolution process specifically called this "swift language mode" to avoid some confusion around terminology, I'd suggest we lean into that and call this `swiftLanguageMode`: https://github.com/swiftlang/swift-evolution/blob/main/proposals/0441-formalize-language-mode-terminology.md

Reply:
Generally, I agree. But from the proposal I don't know, if the Xcode build setting will be renamed or a new one will be added. And I don't know, if the String rawValue of SettingName is used somewhere.

Reply:
I'd also name this `swiftLanguageMode` since that's how this is currently named. This settings relates to SPM, so we should use whatever naming they have.